pkgs/tools/misc/zellij/default.nix

```diff
@@ -3,6 +3,7 @@
 , rustPlatform
 , stdenv
 , installShellFiles
+, perl
```

Comment:
hey, hope you don't mind the comment, was checking if anyone made an upgrade PR and saw yours and was a bit confused and curious as to why Perl was added as a dependency? could not find anything about it in the Zellij repo and hoped you might help me understand how you can figure such things out!

thanks for taking the time to upgrade the package, really looking forward to enjoying the update!

Reply:
Oh, perl is a dependency of openssl (they have their own custom, perl-based configure script) and this release of zellij added openssl as a dependency.
Also, for rust stuff, all dependencies are vendored, so openssl is built as part of the zellij build.

Honestly, I didn't notice this from the release diff either, I just built the new version and the error that I got was pretty clear.

Reply:
Ah I see, nice catch! Thanks for explaining!